### PR TITLE
Solution in C for problem 4 and problem 28

### PR DESCRIPTION
**Pull request by sanchi4**

### CODE REVIEW

**Comment by ryukinix** [comment]
Thanks for your interesting in contributing to our project! I'll take a quick code review in your pull request and we can merge it in soon.

File: Problem004/C/solution_1.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <stdlib.h>
+int main()
+{
+    unsigned long int m=0,n=0, palindrome=0,h;
+     int i=0,j=0;
```
Comment (ryukinix) [inline Problem004/C/solution_1.c]: Please, fix the indentation here.

File: Problem028/C/solution_1.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	for(i=0;i<1001;++i)
+	{
+		for(j=0;j<1001;++j)
+		{
+			if(j==i || j+i==1000)
+			sum=sum+g[i][j];
+		}
+	}
+	printf("%lld",sum);
+    return 0;
```
Comment (ryukinix) [inline Problem028/C/solution_1.c]: Same here, fix indentation.

File: Problem004/C/solution_1.c (hunk 1 above)
Comment (sanchi4) [reply Problem004/C/solution_1.c]: Done

File: Problem028/C/solution_1.c (hunk 2 above)
Comment (sanchi4) [reply Problem028/C/solution_1.c]: Done with editing both of them.

**Pull request closed without merging**